index.js

```diff
@@ -106,6 +106,10 @@ class XiaomiRoborockVacuum {
     this.config.pauseWord = config.pauseWord || "Pause";
     this.config.findMe = config.findMe || false;
     this.config.findMeWord = config.findMeWord || "where are you";
+    this.config.goTo = config.goTo || false;
+    this.config.goToWord = config.goToWord || "time to empty";
```

Comment:
I love your use case, but probably `Go to coordinates` is more generic to everyone :)

Reply:
Good idea. I was out of ideas yesterday evening 😉